creator-node/src/routes/nodeSync.js

```diff
@@ -38,13 +40,16 @@ module.exports = function (app) {
         models.Track.findAll({ where: { cnodeUserUUID: cnodeUserUUIDs }, transaction: t }),
         models.File.findAll({ where: { cnodeUserUUID: cnodeUserUUIDs }, transaction: t })
       ])
-      await t.commit()
+      await t.commit() // why do we have this if we're not making changes?
```

Comment:
i suspect this is here so that we have consistency between reads for cnodeUsers and the models - without the tx we could potentially have a discrepancy. not 100% sure